application/views/js_init/users/js_users.php

```diff
@@ -1,7 +1,7 @@
 <script language="javascript">
 $(document).ready(function() {
 
-var inbox_master = '<?php echo $this->config->item('inbox_owner_id');?>';
+var inbox_master = '<?php echo $this->config->item('inbox_owner_id')[0];?>';
```

Comment:
Why `[0]`th element, has the return type of config changed?

Reply:
in file config/kalkun_settings this variable return array variabel, but in js file that require interger value as id, so we must add index array to get value from array in index 0